.github/workflows/deploy.yml

```diff
@@ -0,0 +1,85 @@
+# If this workflow is triggered by a push to master, it
+#        deploys a SNAPSHOT
+# If this workflow is triggered by publishing a Release, it
+#        deploys a RELEASE with the selected version
+#        updates the project version by incrementing the patch version
+#        commits the version update change to the repository's default branch.
+name: Deploy artifacts with Maven
+on:                                          # yamllint disable-line rule:truthy
+  push:
+    branches: [master]
+  release:
+    types: [published]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Detect release type
+      run: |-
+        if [[ "${{github.event.ref}}" == "refs/heads/master" ]]; then
+          echo "::set-output name=type::SNAPSHOT"
+        else
+          echo "::set-output name=type::RELEASE"
+        fi
+      id: release-detection
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Import Secrets
+      uses: RichiCoder1/vault-action@v0.10.2
+      with:
+        url: ${{ secrets.VAULT_ADDR }}
+        method: approle
+        roleId: ${{ secrets.VAULT_ROLE_ID }}
+        secretId: ${{ secrets.VAULT_SECRET_ID }}
+        secrets: |
+          common/jenkins/main NEXUS_APP_CAMUNDA_COM_PSW | NEXUS_APP_CAMUNDA_COM_PSW;
+          common/jenkins/main NEXUS_APP_CAMUNDA_COM_USR | NEXUS_APP_CAMUNDA_COM_USR;
```

Comment:
Is this account the common `jenkins` shared account we are trying to migrate out of? Or it's a new account that we use for github?

Reply:
that's `ci-cambpm`. I wanted to avoid having to ask IT to create an account.

However, as a preparation we can create a dedicated entry for GitHub Actions.